modules/pattern/pattern.resolver.js

```diff
@@ -29,7 +42,11 @@ const patternMutation = {
 
   deletePattern: async (parent, args) => {
     try {
-      return await patternService.deletePattern(args.id);
+      const pattern = await Pattern.findById(args.id).lean();
+      const deletedImages = await deleteFiles(Object.values(pattern.images));
+      if (await Promise.allSettled(deletedImages)) {
```

Comment:
Why do you need this line if `deletedImages` is an array of values, not promises? Do I miss something?

Reply:
they are promises